src/application.ts

```diff
@@ -104,7 +124,7 @@ export class PocketGatewayApplication extends BootMixin(ServiceMixin(RepositoryM
       50,
       100000,
       0,
-      120000,
+      1200000,
       false,
```

Comment:
Should we move these variables to the top and in uppercase, as they're constants?

Reply:
Yes, please. And let's comment as to what they are.